migrations/20160113172933-replace-usergroup-role.js

```diff
@@ -1,15 +1,13 @@
 'use strict';
 
-var roles = require('../server/constants/roles');
```

Comment:
From what I can tell: All the files in `server/` are now es6 compliant. I don't know how to pass babel-node to sequelize to compile `server/` first. You could require `dist/` and that works.